src/robot/libraries/Telnet.py

```diff
@@ -1180,6 +1215,74 @@ def _check_terminal_emulation(self, terminal_emulation):
         return TerminalEmulator(window_size=self._window_size,
                                 newline=self._newline, encoding=self._encoding)
 
+    def process_rawq(self):
```

Comment:
this is the copy of the process_rawq() from python with two lines (1226 and 1228) changed